File: URI/c++/DDD - 1050.cpp
#include <bits/stdc++.h>
using namespace std;
#define INF             (1e9 + 7)
#define SINF            (1e18)
#define all(a)          a.begin(), a.end()
#define fill(t,v)       memset(t, v, sizeof(t))
#define sz(a)           ((int)(a.size()))
#define LOG2(X)         ((unsigned) (8*sizeof(unsigned long long) - __builtin_clzll((X)) - 1))
#define uIndex(x, v)     (upper_bound(all(v), x) - v.begin())
#define lIndex(x, v)     (lower_bound(all(v), x) - v.begin())
typedef long long ll;
typedef pair<int, int> ii;
typedef vector<int> vi;
typedef vector<vi> vvi;
typedef vector<ii> vii;
typedef vector<vii> vvii;
typedef vector<pair<int, ii>> vpii;
typedef set<int> si;
typedef map<string, int> msi;
typedef vector<string> vs;

int main(){
    map<int, string> list = {
        {61, "Brasilia"},
        {71, "Salvador"},
        {11, "Sao Paulo"},
        {21, "Rio de Janeiro"},
        {32, "Juiz de Fora"},
        {19, "Campinas"},
        {27, "Vitoria"},
        {31, "Belo Horizonte"},
    };
    int num;
    scanf("%d", &num);
    printf("%s\n", (list.count(num)? list[num].c_str(): "DDD nao cadastrado"));
    return 0;
}

File: URI/java/DDD - 1050.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
	public static void main(String[] args) throws IOException {
 		Scanner scan = new Scanner(System.in);
 		int num = scan.nextInt();
 		if(num == 61)
 			System.out.printf("Brasilia%n");
 		else if(num == 71)
 			System.out.printf("Salvador%n");
 		else if(num == 11)
 			System.out.printf("Sao Paulo%n");
 		else if(num == 21)
 			System.out.printf("Rio de Janeiro%n");
 		else if(num == 32)
 			System.out.printf("Juiz de Fora%n");
 		else if(num == 19)
 			System.out.printf("Campinas%n");
 		else if(num == 27)
 			System.out.printf("Vitoria%n");
 		else if(num == 31)
 			System.out.printf("Belo Horizonte%n");
	}
}
